app/services/flashes.js

```diff
@@ -61,11 +68,12 @@ export default Service.extend({
       const { message, preamble = messageTypeToPreamble[type], aboveOverlay } = flash[type];
       const icon = messageTypeToIcon[type];
       const closeButton = messageTypeToCloseButton[type];
+      const autoClose = messageTypeToAutoclose[type];
       const item = { type, message, icon, preamble, closeButton, aboveOverlay };
 
       this.flashes.unshiftObject(item);
 
-      if (!closeButton) this.removeFlash(item);
+      if (autoClose) this.removeFlash(item);
```

Comment:
This change seems to be irrelevant. Could you please revert it (and all related ones)?

Reply:
👍